src/System.Private.ServiceModel/tools/scripts/ConfigHttpsPort.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$FriendlyName = "WCF Bridge - Machine certificate generated by the CertificateManager"
+
```

Comment:
@roncain This will NOT check into the infrastructure. Per discussion this morning, we will keep client only changes on infrastructure.